src/main/java/by/custom/utilcalculator/service/BotFieldsManager.java

```diff
@@ -0,0 +1,93 @@
+package by.custom.utilcalculator.service;
+
+import by.custom.utilcalculator.directory.UserProgress;
+import by.custom.utilcalculator.directory.resources.Command;
+import by.custom.utilcalculator.directory.steps.*;
+
+public class BotFieldsManager {
```

Comment:
It's not call to action
```suggestion
public class UserAnswerManager {
```

Reply:
Renamed to UserProgressManager